pubsub/aws/snssqs/metadata.yaml

```diff
@@ -0,0 +1,155 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: aws.snssqs
+version: v1
+status: stable
+title: "AWS SNS/SQS"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-aws-snssqs/
+capabilities:
+  - ttl
+builtinAuthenticationProfiles:
+  - name: "aws"
+metadata:
+  - name: region
+    required: true
+    description: |
+      The AWS region to the instance. See this page for valid regions:
```

Comment:
Same feedback as other PR.

1. Please use the `url` property for URLs
2. "The AWS region to the instance" sounds odd
3. Alos, "Ensure that DynamoDB are available in that region" ... that seems like a typo?